indico/htdocs/sass/_base.scss

```diff
@@ -15,10 +15,11 @@
  * along with Indico; if not, see <http://www.gnu.org/licenses/>.
  */
 
-@import 'base/palette';
-@import 'base/defaults';
-@import 'base/layout';
-@import 'base/reset';
-@import 'base/theme';
-@import 'base/typography';
-@import 'base/utilities';
+@import "base/palette";
```

Comment:
No clue and it works, so I am not touching sass imports because otherwise: :boom: